File: auctionwebapp.client/src/app/bid.ts
import { Lot } from './lot';
import { User } from './user';

export class Bid {
  constructor(
    public BId1?: number,
    public BTime?: Date,
    public BParticipantId?: number,
    public BSize?: number,
    public BLotId?: number,
    public BLot?: Lot,
    public BParticipant?: User
  ) { }
}

File: AuctionWebApp.Server/Data/Models/Bid.cs
using System;
using System.Collections.Generic;

namespace AuctionWebApp.Server.Data.Entities;

public partial class Bid
{
    public ulong BId1 { get; set; }

    public DateTime BTime { get; set; }

    public ulong BParticipantId { get; set; }

    public ulong BSize { get; set; }

    public ulong BLotId { get; set; }

    public virtual Lot BLot { get; set; } = null!;

    public virtual User BParticipant { get; set; } = null!;
}
